.gitignore

```diff
@@ -3,6 +3,8 @@ out
 combined-xml
 *.swp
 *.csv
+!codelist_rules.csv
```

Comment:
I wonder if this line is a typo, and the plan here was in fact to gitignore codelist_rules.json?